drivers/tty/serial/8250/8250_dma.c

```diff
@@ -255,11 +270,13 @@ void serial8250_release_dma(struct uart_8250_port *p)
 
 	/* Release TX resources */
 	dmaengine_terminate_sync(dma->txchan);
-	dma_unmap_single(dma->txchan->device->dev, dma->tx_addr,
-			 UART_XMIT_SIZE, DMA_TO_DEVICE);
+	if (dma->tx_running) {
```

Comment:
Why this check? Termination above is synchronous, should stop DMA.

Reply:
I see. Will remove the check.